### ISSUE TITLE
Terminal misbehaving when editor is called

### ISSUE THREAD

**Issue by pazz** [pazz/alot#1142]
The series in #1135 introduces a strange and unexpected behaviour at my end:
When I reply to a mail and my editor is takes over the terminal, it seems that the keypresses do not reach the editor (vim).

I'm on 3ad6ebf3, the first commit which shows this behaviour.
The alot log sais this:
```
INFO:envelope:editable headers: set([u'Subject'])
DEBUG:globals:using editor_cmd: /usr/bin/vim -f -c 'set filetype=mail' +
DEBUG:globals:{'spawn: ': False, 'in_thread': False}
DEBUG:globals:{'spawn': False}
DEBUG:globals:cmdlist: ['/usr/bin/vim', '-f', '-c', 'set filetype=mail', '+', '/tmp/alot.hZVNjZ.eml']
INFO:globals:calling external command: ['/usr/bin/vim', '-f', '-c', 'set filetype=mail', '+', '/tmp/alot.hZVNjZ.eml']
```

My alot config contains this as the only editor-related setting (all others are commented out):
```
editor_cmd = /usr/bin/vim -f -c 'set filetype=mail' +
```

**Comment by dcbaker** [comment]
Is this a duplicate of #1137, which is fixed by #1138 (I think)?

**Comment by pazz** [comment]
yes, you are right: 222017927663f41 fixes this problem for me.
I'll close this here. Let's push your patch asap.
